boards/entrypoint.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -e 
+
+if [ -z "$AZURE_BOARDS_ORGANIZATION" ]; then
+    echo "AZURE_BOARDS_ORGANIZATION is not set." >&2
+    exit 1
+fi
+
+if [ -z "$AZURE_BOARDS_PROJECT" ]; then
+    echo "AZURE_BOARDS_PROJECT is not set." >&2
+    exit 1
+fi
+
+if [ -z "$AZURE_BOARDS_TOKEN" ]; then
+    echo "AZURE_BOARDS_TOKEN is not set." >&2
+    exit 1
+fi
+
+if [ -z "$GITHUB_EVENT_PATH" ]; then
+    echo "GITHUB_EVENT_PATH is not set." >&2
+    exit 1
+fi
+
+function work_items_for_issue {
+    vsts work item query --wiql "SELECT ID FROM workitems WHERE [System.Tags] CONTAINS 'GitHub' AND [System.Tags] CONTAINS 'Issue ${GITHUB_ISSUE_NUMBER}'" | jq '.[].id' | xargs
+}
+
+AZURE_BOARDS_TYPE="${AZURE_BOARDS_TYPE:-Feature}"
+AZURE_BOARDS_CLOSED_STATE="${AZURE_BOARDS_CLOSED_STATE:-Closed}"
+AZURE_BOARDS_REOPENED_STATE="${AZURE_BOARDS_REOPENED_STATE:-New}"
+
+AZURE_DEVOPS_URL="https://dev.azure.com/${AZURE_BOARDS_ORGANIZATION}/"
+vsts configure --defaults instance="${AZURE_DEVOPS_URL}" project="${AZURE_BOARDS_PROJECT}"
+
+vsts login --token ${AZURE_BOARDS_TOKEN}
```

Comment:
This should be quoted; `"${AZURE_BOARDS_TOKEN}"`

Reply:
Fixed